randovania/games/factorio/gui/dialog/game_export_dialog.py

```diff
@@ -11,14 +12,28 @@
 from randovania.gui.dialog.game_export_dialog import (
     GameExportDialog,
     add_field_validation,
+    is_directory_validator,
     prompt_for_output_directory,
     spoiler_path_for_directory,
 )
+from randovania.lib import windows_lib
 
 if TYPE_CHECKING:
     from randovania.interface_common.options import Options, PerGameOptions
 
 
+def get_global_user_data_dir() -> Path:
+    match platform.system():
+        case "Windows":
+            return windows_lib.get_appdata().joinpath("Factorio")
+        case "Darwin":
+            return Path("~/Library/Application Support/factorio")
+        case "Linux":
+            return Path("~/.factorio")
```

Comment:
*surely* factorio respects the XDG spec?

Reply:
I'm just following their wiki https://wiki.factorio.com/Application_directory

Reply:
curse you factorio